### DISCUSSION TITLE
What does the instance score mean?

### DISCUSSION THREAD

**Discussion by msjindal** [talmolab/sleap#1680] [Help!]
I work in a lab that handles tracking rats to determine dominance and social behavior. I was curious when outputting the data it provides me an instance score. What does that exactly mean? What is it calculating?

**Comment by Marielamend** [comment]
Hello @msjindal!

This past discussion is relevant to your question and might answer other questions regarding accuracy scores! #1268 
> Per the doc strings, the instance score is supposed to measure the grouping of nodes.
[sleap/sleap/instance.py](https://github.com/talmolab/sleap/blob/b5aab0edfbcfddce82e8ee02d9bd383604b9de93/sleap/instance.py#L1166-L1167)

             instance_score: Scalar float representing the overall instance score, e.g., 
                 the PAF grouping score.

Any other questions let us know!
Best,
Mariela

**Comment by roomrys** [comment]
Hi @msjindal,

I saw your new question in #1682 and just wanted to follow-up here as well since they were related (but different enouugh).

From #1268
> the instance score is supposed to measure the grouping of nodes

So, the instance score _should_ measure how confident SLEAP is that the nodes grouped together in an `Instance` should be grouped together in that `Instance`. Instance scores are used to solve the grouping problem: "should `Node X` belong to `Instance A` or `Instance B`?".

Now, depending on which model the user elects to use, how the instance score is calculated varies (simply due to the nature of how each model handles the part grouping problem differently). This is detailed in #1268.

Thanks,
Liezl
